src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  blocktime: 4,
```

Comment:
Make this configurable either via an `.env` file or autoconfigure it via API.

Reply:
https://github.com/ArkEcosystem/poc-ark-messenger/pull/7